es/README.md

```diff
@@ -4,19 +4,19 @@
 
 > Este trabajo está bajo la licencia internacional Creative Commons Attribution-ShareAlike 4.0. Para ver una copia de esta licencia, visita el siguiente enlace https://creativecommons.org/licenses/by-sa/4.0/
 
-## Bienvenida/o
+## Bienvenido/a
```

Comment:
@magul here I am not sure, cause since the workshop is targeted at women, the original translation incuded

Welcome (female) / Welcome (male)

This PR inverts the order

Welcome (male) / Welcome (female)

I am not sure this is the right approach or even the correct use of spanish (using /o or /a at the end)

So we could:

`Bienvenidos` -> Plural, no gender association.
`Te damos la bienvenida`  -> Addresses a single person and no gender association